frontend/src/components/AlertForm.js

```diff
@@ -0,0 +1,36 @@
+//import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
+
+export default function AlertForm() {
+    const [inputValue, setInputValue] = useState('');
+    const handleInput = (e) => {
+        const formattedPhoneNumber = formatPhoneNumber(e.target.value);
+        setInputValue(formattedPhoneNumber);
+    };
+    
+    return (
+        <form>
+            <label>
+                Email:
+                <input type="email" name="email" />
+            </label>
+            <label>
+                Phone Number:
+                <input onChange={e => handleInput(e)} value={inputValue} />
+            </label>
+            <input type="submit" value="Sign up" />
+        </form>
+    )
+}
```

Comment:
This is good for now, but for the future I'd recommend trying out the bootstrap form/label components for styling